migrations/sql/V2019.02.07.02.27__alter_expected_document_status.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE mine_expected_document DROP CONSTRAINT mine_expected_document_exp_document_status_guid_fkey;
+ALTER TABLE mine_expected_document DROP COLUMN exp_document_status_guid;
+ALTER TABLE mine_expected_document ADD COLUMN exp_document_status_code character varying(3) NOT NULL DEFAULT 'MIA';
+
+DROP TABLE mine_expected_document_status;
+CREATE TABLE mine_expected_document_status_code (
+    exp_document_status_code character varying(3) PRIMARY KEY DEFAULT 'MIA',
+    description character varying(100) NOT NULL,
+    display_order smallint NOT NULL,
```

Comment:
minor thought... would it make sense to have display_order as an optional field? In the case where a code is inactive, would there still be a display order value?

Reply:
Good point, thats probably a good change for maximum future compatability